beta/experience-platform-extension/tutorials/README.md

```diff
@@ -25,3 +25,21 @@ Get started with [Assignment 1 - AEP Edge extension setup and XDM usage](https:/
 * [ ] XDM implementation validation
 * [ ] Clarity of documentation during the beta review call
 
+
+### Assignment 2
+
+In this assignment you will learn how to:
+
+1. Install Swfit Core library using Cocoapods
+2. Initialize SDK with new API
+3. Start/Stop lifecycle
+4. Make an Identity call
+
+Get started with [Assignment 1 - AEP Edge extension setup and XDM usage](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-1-edge-extension-setup).
```

Comment:
curious about the link to assignment 1 here, since technically all the steps in this assignment were done already for that one